### PR TITLE
Create main.yml

### PR DESCRIPTION
**Pull request by DM0000**

### CODE REVIEW

File: src/main.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import "bootstrap/dist/js/bootstrap.bundle.js";
 import startServer from "./server/fakeServer.jsx";
 
 startServer();
-
+//ffffff
```
Comment (rdkerr) [inline src/main.jsx]: ?

File: package.json [hunk 2]
```diff
@@ -10,16 +10,15 @@
...
   "dependencies": {
-    "@faker-js/faker": "^9.0.0",
-    "bootstrap": "^5.3.3",
-    "husky": "^9.1.6",
     "prop-types": "^15.8.1",
     "react": "^18.3.1",
     "react-bootstrap": "^2.10.5",
-    "react-dom": "^18.3.1"
+    "react-dom": "^18.3.1",
+    "save": "^2.9.0"
```
Comment (rdkerr) [inline package.json:19]: Whats this used for?

**Comment by rdkerr** [comment]
merge conflict

File: package.json [hunk 3]
```diff
@@ -10,16 +10,15 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@faker-js/faker": "^9.0.0",
-    "bootstrap": "^5.3.3",
-    "husky": "^9.1.6",
```
Comment (rdkerr) [inline package.json]: You can remove this if you want, there are some useful hooks you can add with this.  Like format on commit so you don't have to worry about fixing eslint stuff

Comment (rdkerr) [reply package.json]: You have a husky hook in yoru package.json. Probably shouldn't remove this

File: src/main.jsx (hunk 1 above)
Comment (DM0000) [reply src/main.jsx]: messing with stuff  / maybe ben

File: package.json (hunk 2 above)
Comment (DM0000) [reply package.json:19]: react-dom used to escape react if needed(not being used currently) installed it with react bundle at start before started merging react-bootstrap

File: package.json (hunk 3 above)
Comment (DM0000) [reply package.json]: removed when fightring with eslint action
readded

File: package.json (hunk 2 above)
Comment (rdkerr) [reply package.json:19]: I meant save

**Pull request merged**